src/config/ePoint.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+import Config from 'react-native-config';
+
+const search = axios.create({
+  baseURL: Config.BACKEND_URL,
+  headers: {
+    'User-Agent': Config.USER_AGENT,
+  },
+});
+
+export default search;
```

Comment:
we dont need to add new api file for that, because we are using same backend url in here. My suggestion is

const headers: {
    'User-Agent': Config.USER_AGENT,
  }
ePointApi.post('/usr-activity', params, {headers: headers})

we can use this code inside of ePoint provider